services/document-manager/backend/app/__init__.py

```diff
@@ -30,6 +30,9 @@ def create_app(test_config=None):
         dictConfig(Config.LOGGING_DICT_CONFIG)
     app = Flask(__name__)
 
+    if not os.path.exists('/tmp/spatial'):
+        os.makedirs('/tmp/spatial')
```

Comment:
![image](https://github.com/user-attachments/assets/88a73ee7-2ffe-4d01-bdfa-ad7ed65a984a)
Looks like this doesn't match exactly with the path the files are downloaded to?
Think we might need to add a leading `/` there